wpimath/src/main/native/include/frc/system/plant/LinearSystemId.h

```diff
@@ -96,6 +118,9 @@ class LinearSystemId {
    * Constructs the state-space model for a 1 DOF position system from system
    * identification data.
    *
+   * You MUST use an SI unit (i.e. meters or radians) for the Distance template
+   * argument.
```

Comment:
The units of Kv and Ka can be something else like imperial though. They'll get converted to the the units of the template argument.

Reply:
Yes, the user can provide anything for their argument (i.e. 1_V / 3_fps), but the `Distance` template argument must be `units::meter` or `units::radian` for it to get converted to SI internally.

Reply:
Of course. I just want to avoid a team misunderstanding this comment and going "reeee, don't take away mah freedom units!"